nailgun/entities.py

```diff
@@ -952,6 +952,38 @@ def copy(self, name):
         response.raise_for_status()
         return response.json()
 
+    def delete_from_environment(self, environment, synchronous=True):
+        """Delete this content view version from an environment.
+
+        This method acts much like
+        :meth:`nailgun.entity_mixins.EntityDeleteMixin.delete`.  The
+        documentation on that method describes how the deletion procedure works
+        in general. This method differs only in accepting an ``environment``
+        parameter.
+
+        :param environment: A :class:`nailgun.entities.Environment` object. The
+        environment's ``id`` parameter *must* be specified. As a convenience,
+        an environment ID may be passed in instead of an ``Environment``
+        object.
```

Comment:
:param environment: A :class:`nailgun.entities.Environment` object. The
        environment's ``id`` parameter *must* be specified. As a
        convenience, an environment ID may be passed in instead of an
        ``Environment`` object.